concrete/src/Database/DatabaseManagerORM.php

```diff
@@ -21,16 +19,18 @@ class DatabaseManagerORM
     /**
      * Create a new database ORM manager instance.
      *
-     * @param Application $app
+     * @param \Concrete\Core\Application\Application $app
```

Comment:
I'm not sure I understand the purpose of changing Application to the FQN, when Application is being imported already.